generator/testdata/googleapis/README.md

```diff
@@ -21,4 +21,15 @@ cp $HOME/googleapis/google/iam/v1/options.proto    ./google/iam/v1
 cp $HOME/googleapis/google/cloud/secretmanager/v1/resources.proto        ./google/cloud/secretmanager/v1
 cp $HOME/googleapis/google/cloud/secretmanager/v1/secretmanager_v1.yaml  ./google/cloud/secretmanager/v1
 cp $HOME/googleapis/google/cloud/secretmanager/v1/service.proto          ./google/cloud/secretmanager/v1
+cp $HOME/googleapis/google/cloud/sql/v1/cloud_sql_backup_runs.proto          ./google/cloud/sql/v1
+cp $HOME/googleapis/google/cloud/sql/v1/cloud_sql_connect.proto          ./google/cloud/sql/v1
+cp $HOME/googleapis/google/cloud/sql/v1/cloud_sql_databases.proto          ./google/cloud/sql/v1
+cp $HOME/googleapis/google/cloud/sql/v1/cloud_sql_flags.proto          ./google/cloud/sql/v1
+cp $HOME/googleapis/google/cloud/sql/v1/cloud_sql_instances.proto          ./google/cloud/sql/v1
+cp $HOME/googleapis/google/cloud/sql/v1/cloud_sql_operations.proto          ./google/cloud/sql/v1
+cp $HOME/googleapis/google/cloud/sql/v1/cloud_sql_ssl_certs.proto          ./google/cloud/sql/v1
+cp $HOME/googleapis/google/cloud/sql/v1/cloud_sql_tiers.proto          ./google/cloud/sql/v1
+cp $HOME/googleapis/google/cloud/sql/v1/cloud_sql_users.proto          ./google/cloud/sql/v1
+cp $HOME/googleapis/google/cloud/sql/v1/sqladmin_v1.yaml          ./google/cloud/sql/v1
+cp $HOME/googleapis/google/cloud/sql/v1/cloud_sql_resources.proto          ./google/cloud/sql/v1
```

Comment:
Do we need these files?  Effectively your integration tests are doing the same thing, no?